data/modules/SearchRescue/SearchRescue.lua

```diff
@@ -2120,7 +2122,7 @@ local buildMissionDescription = function(mission)
 			l.LON.." "..decToDegMinSec(math.rad2deg(mission.long))
 	end
 
-	local shipname = ShipDef[mission.shipid].name
+	local shipname = ShipDef[mission.shipid].name -- local shipname = ls[ShipDef[mission.shipid].modelName]
```

Comment:
... what is the meaning of this line?

Reply:
Looks like commented out experimental code.